pyne/partisn.py

```diff
@@ -529,7 +529,7 @@ def _write_input(title, block01, block02, block03, block04, block05, cards, file
 
     block1_cards = []
     if 'block1' in cards:
-        for card, value in cards['block1'].iteritems():
+        for card, value in sorted(cards['block1'].items()):
```

Comment:
Why sorted here? Should we be using a different data structure?

Reply:
Sort the content here is to make sure the output blocks have the same sequence in python2 and python3. Otherwise, they have different sequences, then the output file compare will return a `False`.

Reply:
I'm wondering if using an OrderedDict might be useful here.